io.github.zen_browser.zen.yml

```diff
@@ -0,0 +1,46 @@
+app-id: io.github.zen_browser.zen
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '23.08'
+command: launch-script.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=cups
+  - --persist=.zen
+  - --filesystem=xdg-download:rw
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+modules:
+  - name: zen_browser
+    buildsystem: simple
+    build-commands:
+      - mv zen/* /app/
+
+      - install -d /app/share/icons/hicolor/scalable/apps/
+
+      - cp -r icons/* /app/share/icons/hicolor/scalable/apps/
```

Comment:
There is no icon inside the archive. This is copying the updater icon.

Reply:
There is under icons/